gitops/environment/plnsvc/base/argocd-rbac/argo-syncer-role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: argocd-manager-role
+  namespace: kcp-syncer
```

Comment:
I am not following. I feel like it is reasonable to have a kcp component named syncer deployed under a namespace named kcp-syncer.

`kube-system` is definitely a bad choice.

Kubernetes documentation: "kube-system The namespace for objects created by the Kubernetes system"

Quoting David Eads: "kube-system contains service accounts which are used to run the kubernetes controllers. These service accounts are granted significant permissions (create pods anywhere, for instance). Since openshift builds on top of kube, we inherit the structure.
You should avoid putting anything "personal" in that namespace since kube considers it to be "owned" by kube and the permissions for the SAs inside are quite high"

`plnsvc` seems like a catch-all thing

Reply:
yes that's correct. It is also what the argocd client generates when namespaces are specified with `argocd cluster add`